benchmarks/distributed/rpc/rl/agent.py

```diff
@@ -0,0 +1,150 @@
+from functools import reduce
```

Comment:
since files in `benchmarks/distributed/rpc` will be user-facing, we are going to need comprehensive comments to explain what each class/function is doing.